### ISSUE TITLE
Seems that SQL LIKE Wildcard not implement

### ISSUE THREAD

**Issue by SyuTingSong** [catfan/Medoo#323]
```
$database->select("person", "id", [
    "city[~]" => "stan%" // Kazakhstan,  Uzbekistan, Türkmenistan
]);
```

Gets the SQL:

```
SELECT "id" FROM "person" WHERE "city" LIKE 'stan%'
```

As the doc spec should be:

```
SELECT "id" FROM "person" WHERE "city" LIKE '%stan'
```

---

```
$database->select("person", "id", [
    "city[~]" => "Londo_" // London, Londox, Londos...
]);
```

Gets the SQL:

```
SELECT "id" FROM "person" WHERE "city" LIKE '%Londo_%'
```

As the doc spec should be:

```
SELECT "id" FROM "person" WHERE "city" LIKE 'Londo_'
```

**Comment by glenndavey83** [comment]
I've added support for SQL LIKE Wildcards as a fix for this issue in my pull request #328. 

I think it's important to implement sooner rather than later if the online Docs list this feature. More testing and optimization may be done later for edge cases.

**Comment by SyuTingSong** [comment]
Thank you @glenndavey83 I found this issue when I am writing test-cases for Medoo. I will merge your code to my fork and test it.

To @catfan: I read some docs for MySQL, PostgreSQL, MS SQL Server, Oracle, SQLite and Sybase. The Syntax `[cbr]at` for matching `cat`, `bat`, `rat` is on originally supported by MS SQL Server. And it supports `[^cbr]at` for matching any 3-letters word end with `at` and not beginning with `c`, `b` or `r`.

Medoo should either update the documentation or implement this in PHP.
